### ISSUE TITLE
SimpleAllele.alleleName.name is not documented

### ISSUE THREAD

**Issue by alexhenrie** [tnavatar/clingen-data-model#145]
http://datamodel.clinicalgenome.org/main/resources/clingen-xsd/simpleallele.xsd defines "name" as a property of the "SimpleAllele.AlleleName" type, but it is not listed under http://datamodel.clinicalgenome.org/allele/resource/simple_allele/#simplealleleallelename

**Comment by larrybabb** [comment]
sorry for the delayed response.

first a note: we have just recently changed the class name SimpleAllele to AlleleInstance.  The qualifier "simple" is being used primarily to distinguish between simple and complex CanonicalAlleles and having it embedded in the separate class name was creating confusion.

The alleleName property on AlleleInstance (frmlry SimpleAllele) is an association to the AlleleInstanceName secondary/dependent resource.  This is indicated by the property type of "AlleleInstance.AlleleName" in the [alleleinstance.xsd](http://datamodel.clinicalgenome.org/main/resources/clingen-xsd/alleleinstance.xsd).

The documentation does have a heading for AlleleInstance.alleleName with a definition. It is a subheading which contains additional attribute/field definitions so it may not seem like a simple attribute of the main AlleleInstance class.
http://datamodel.clinicalgenome.org/allele/resource/allele_instance/#alleleinstanceallelename

Please clarify if I missed your point.

**Comment by alexhenrie** [comment]
Hi Larry, thanks for the reply. Let me try again to explain: According to [the documentation](http://datamodel.clinicalgenome.org/allele/resource/allele_instance/#alleleinstanceallelename), AlleleInstance.alleleName represents "The name given to the allele by a naming organization (i.e. an HGVS expression)". But the documentation defines AlleleInstance.alleleName as an object with three properties: nameType, legacy, and preferred. The HGVS expression does not belong in any of these properties. Instead, there is another, undocumented, property for the HGVS expression: AlleleInstance.alleleName.name.

I'm asking you to add AlleleInstance.alleleName.name to the documentation and clarify that the HGVS expression belongs in this property of the AlleleNameInstance class.
